<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EcuVersionForm</class>
 <widget class="QWidget" name="EcuVersionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pushButton_del{
	background-color: rgb(78, 154, 6);
color:white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>protocol:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_firmware"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_software"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_protocol"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>manufacturer:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>firmware:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_manufacturer"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>software:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ecu:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_ecu">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Unknow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tbox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mcu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dcdc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Abs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Scm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inv</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ofbc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Imu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tft</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lcsc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rcsc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Flm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hlc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wpc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dik</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lpdc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tpms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cddc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gcu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vcu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hmi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Obc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hhd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Obd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TboxSK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BmsSK</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_del">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
